pkg/cronjob/es_rollover.go

```diff
@@ -125,13 +125,14 @@ func createTemplate(name, action string, jaeger *v1.Jaeger, envs []corev1.EnvVar
 			Volumes:            commonSpec.Volumes,
 			Containers: []corev1.Container{
 				{
-					Name:         name,
-					Image:        util.ImageName(jaeger.Spec.Storage.EsRollover.Image, "jaeger-es-rollover-image"),
-					Args:         []string{action, util.GetEsHostname(jaeger.Spec.Storage.Options.Map())},
-					Env:          util.RemoveEmptyVars(envs),
-					EnvFrom:      envFromSource,
-					Resources:    commonSpec.Resources,
-					VolumeMounts: commonSpec.VolumeMounts,
+					Name:            name,
+					Image:           util.ImageName(jaeger.Spec.Storage.EsRollover.Image, "jaeger-es-rollover-image"),
+					Args:            []string{action, util.GetEsHostname(jaeger.Spec.Storage.Options.Map())},
+					Env:             util.RemoveEmptyVars(envs),
+					EnvFrom:         envFromSource,
+					Resources:       commonSpec.Resources,
+					VolumeMounts:    commonSpec.VolumeMounts,
+					SecurityContext: commonSpec.ContainerSecurityContext,
```

Comment:
Does this have a default value? what happens if it's not set in the CR?

Reply:
Hi @rubenvp8510 , 
securityContext will be `{}` (empty)
The same here: https://github.com/jaegertracing/jaeger-operator/blob/2b0af44fc671da6d6eb0126e8eb78999beb09df6/pkg/deployment/agent.go#L196C60-L196C60

Reply:
Make sense! Thanks!